app/import_functions.go

```diff
@@ -889,7 +891,7 @@ func (a *App) ImportPost(data *PostImportData, dryRun bool) *model.AppError {
 		if err != nil {
 			return err
 		}
-		post.FileIds = fileIds
+		post.FileIds = append(post.FileIds, fileIds...)
```

Comment:
It makes the attachments from multiple exports additive (Up to 5, then the attachments will be discarded).

Reply:
Can't find the logic where we discard file attachments of more than 5.  Could you please clarify?

Reply:
Actually is not restricted here (and the file is uploaded and bound to the post), is restricted by the application elsewhere. I going to check where.

Reply:
It is checked in the `IsValid` method of the `model.Post` object. Actually it logs errors when it happens.

Reply:
Oh, yeah, that's weird.  So in server it's limited by `POST_FILEIDS_MAX_RUNES = 150` or `POST_FILENAMES_MAX_RUNES    = 4000` which is kind of indirect.  But on webapp, it's straight forward by `MAX_UPLOAD_FILES = 5`

Reply:
Yes, that is weird, but actually POST_FILEIDS_MAX_RUNES = 150 is equivalent to MAX_UPLOAD_FILES = 5 in this case, because the file ids are always 26 characters, and it is serialized in in JSON, so the 5 files length is a string of 146 characters.

Reply:
Is the limitation purely an artifact of the column type used to represent the associated file ids? I'm curious why this isn't a separate table. It seems odd to throw away attachments on import, though I realize this is out of scope for this PR.